superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -229,6 +230,7 @@ class SliceHeaderControls extends React.PureComponent<
         value.behaviors?.includes(Behavior.INTERACTIVE_CHART),
       )
       .find(([key]) => key === slice.viz_type);
+    const emitCrossFilter = slice.form_data?.emit_filter;
```

Comment:
Can we name it sth like `canEmitCrossFilter`? It might be mistaken for a function that emits a cross filter

Reply:
Yes. Changed.